dali/kernels/CMakeLists.txt

```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-project(kernels)
+project(${dali_kernel_lib})
```

Comment:
These names are intended as target names - please keep the project name literal, so those who maintain this code don't have to chase definitions.